tests/e2e/utils/load_balancer/setup_load_balancer.py

```diff
@@ -91,11 +92,20 @@ def create_certificates(
         )
         input("Press any key once the certificate status is ISSUED")
 
-    subdomain_cert_deployment_region = AcmCertificate(
-        domain="*." + subdomain_hosted_zone.domain,
-        hosted_zone=subdomain_hosted_zone,
-        region=deployment_region,
-    )
+    args = {
+        "domain": "*." + subdomain_hosted_zone.domain,
+        "hosted_zone": subdomain_hosted_zone,
+        "region": deployment_region,
+    }
+
+    if subdomain_subject_alternative_names:
```

Comment:
can you explain what a subdomain_subject_alternative_name is ?

Reply:
https://awslabs.github.io/kubeflow-manifests/docs/component-guides/kserve/tutorial/#request-a-certificate